tests/data/test_resample.py

```diff
@@ -479,6 +480,101 @@ def test_skip_minutes_multiple(self, field):
                         asset, field, minute))
 
 
+class MinuteToDailyAggregationFuturesTestCase(WithBcolzFutureMinuteBarReader,
+                                              ZiplineTestCase):
+    #    March 2016
+    # Su Mo Tu We Th Fr Sa
+    #        1  2  3  4  5
+    #  6  7  8  9 10 11 12
+    # 13 14 15 16 17 18 19
+    # 20 21 22 23 24 25 26
+    # 27 28 29 30 31
+
+    TRADING_CALENDAR_STRS = ('us_futures',)
+    TRADING_CALENDAR_PRIMARY_CAL = 'us_futures'
+
+    TRADING_ENV_MIN_DATE = START_DATE = pd.Timestamp(
+        '2016-03-01', tz='UTC',
+    )
+    TRADING_ENV_MAX_DATE = END_DATE = pd.Timestamp(
```

Comment:
I don't think `TRADING_ENV_MIN_DATE` and `TRADING_ENV_MAX_DATE` are necessary here (the test works without them)